src/create_disk.sh

```diff
@@ -307,23 +307,37 @@ install_uefi() {
     # change our build process to download+extract it separately.
     local source_efidir="${deploy_root}/usr/lib/ostree-boot/efi"
     local target_efi="$rootfs/boot/efi"
-    local target_efiboot="${target_efi}/EFI/BOOT"
-    mkdir -p "${target_efiboot}"
-    /usr/lib/coreos-assembler/cp-reflink "${source_efidir}/EFI/BOOT/BOOT"* "${target_efiboot}"
-    local src_grubefi=$(find "${source_efidir}"/EFI/ -name 'grub*.efi')
-    /usr/lib/coreos-assembler/cp-reflink "${src_grubefi}" "${target_efiboot}"
+    local src_grubefi=$(find "${source_efidir}"/EFI/ -maxdepth 1 -type d | grep -v BOOT)
+    local vendor_id="${src_grubefi##*/}"
+    local vendordir="${target_efi}/EFI/${vendor_id}"
+
+    # Some of the files in EFI/BOOT are _symlinks_ to EFI/$VENDOR
+    # in the OS tree. We need to make copies here.
```

Comment:
Seems like it's just `grubenv` that's a symlink; may be easier to special case that?

Also worth commenting here probably are
https://github.com/coreos/rpm-ostree/pull/969
and
https://github.com/ostreedev/ostree/pull/1873

which may be related to some of this being awkward.

In fact we should probably change coreos-assembler (and/or rpm-ostree) to download the grub/efi bits *separately* and not commit them to the ostree as discussed.  But, definitely that can come later.